assets/component-image-with-text.css

```diff
@@ -311,10 +311,14 @@
 }
 
 .image-with-text__heading {
+  word-break: break-word;
+  hyphens: auto;
```

Comment:
The one thing I'd say is that the `hyphens` property isn't used across the theme so it might feel a bit different here compared to other parts/sections of the theme.

Reply:
Yeah I think it gives a better visual cue that it's the same word. I know in french it makes to see/use it. Not sure for other languages tho 🤔

Reply:
Thanks for the tag @melissaperreault! Agreed adding a hyphen is an improvement for now. I'd love to be involved when we pick this up later - seems like there is a lot of intricacies based on the language being used.

Reply:
but then maybe we add `hyphens` in the 14 other places we use `word-break` was my thought 🤔  So that we don't have an hyphen on image with text but not on say image banner 🤔 
Would be a quick thing to add. Im not married to the idea but I think having the same behaviour everywhere would be a plus

Reply:
@ludoboludo  is there a way for me to see how the hyphenation works? I don't currently see it in the editor link above.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens

This is the documentation for hyphens in CSS. We would be using `hyphens: auto` in this solution

Reply:
> Hyphenation rules are language-specific. In HTML, the language is determined by the lang attribute, and browsers will hyphenate only if this attribute is present and the appropriate hyphenation dictionary is available. In XML, the xml:lang attribute must be used.

Looks like the CSS property accounts for language based hyphenation, so since our `html` tag does use the `lang` attribute, we might get baked in i18n support.

Reply:
Looks like support is a bit [hit or miss depending on the language](https://screenshot.click/22-01-u3sn9-p0auh.png).

Here is where I see it working as expected @katycobb: [screenshot](https://screenshot.click/22-01-qinj9-mopuo.png). I had to resize my browser window a bit to have it come up.

Though it does not come up on the title for some reasons. 🤔  I tried changing the word too but no hyphen

Reply:
> Though it does not come up on the title for some reasons. 🤔 I tried changing the word too but no hyphen

It doesn't because some font don't _support_ it ([Video reference](https://screenshot.click/22-01-t5nrj-15hww.mp4))